tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "noUnusedLocals": true,
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
-        "sourceMap": false,
+        "sourceMap": true,
```

Comment:
I confirmed w/ @fragosti that this is cool, and is already being done in https://github.com/0xProject/0x-api/pull/168

Reply:
I did this so I can get proper TS-aware stack traces when tests fail